src/govuk/helpers/_typography.scss

```diff
@@ -167,11 +171,12 @@
 
 /// Font helper
 ///
-/// @param {Number} $size - Size of the font as it would appear on desktop -
-///   uses the responsive font size map
+/// @param {Number} $size Point from the spacing scale (the size as it would
+///   appear on tablet and above)
 /// @param {String} $weight [regular] - Weight: `bold` or `regular`
 /// @param {Boolean} $tabular [false] - Whether to use tabular numbers or not
-/// @param {Number} $line-height [false] - Line-height, if overriding the default
+/// @param {Number} $line-height [false] - Line-height, if overriding the
+///   default
 ///
```

Comment:
Do we want a `@throw` here or is `govuk-font` expected to fail silently? _Should_ it fail silently if so?

Reply:
I've pushed [a change](https://github.com/alphagov/govuk-frontend/compare/42e46e13fd619b33d33c0b54cf68182c17e19462..9ffed11085ed9509bfd80a377f54e1b83d22e4de) that adds a `@throw` annotation to the `govuk-font` mixin as suggested.